### ISSUE TITLE
GStreamer-based ReplayGain Plugin Initialisation Fails

### ISSUE THREAD

**Issue by KingJ** [linuxserver/docker-beets#20]
Host OS: CoreOS 1235.4.0.
docker-beets version tag: 52

I have configured the beets replaygain plugins as follows;

```
replaygain:
    backend: gstreamer
    auto: yes
```

However, when running any `beet` commands, I receive the following error;

```
root@beets-709324527-mbbqf:/$ beet ls
error: replaygain initialization failed: Failed to load GStreamer: python-gi not found
```

Checking the Dockerfile, python-gi should be installed as the py-gobject package has been installed. However, this appears to be version 2.0, whereas the [beets documentation](http://beets.readthedocs.io/en/latest/plugins/replaygain.html#gstreamer) mentions version 3.x. I did attempt to remove the py-gobject package and install py-gobject3 but to no avail - beets returns the following error instead;

```
root@beets-709324527-mbbqf:/$ beet ls
error: replaygain initialization failed: Failed to load required GStreamer plugins
```

Do you know what changes need to be made in order to make use of the GStreamer-based ReplayGain plugin?

Many thanks.

**Comment by KingJ** [comment]
I noticed that version 53 was pushed to the Docker Hub, however I can confirm that the issue is still present on that version.

**Comment by KingJ** [comment]
I might have actually figured this out. Although installing the `py-gobject3` package allows beets to proceed further, it is still missing something. Diving over some of the beets code, and issues such as beetbox/beets/issues/1830 I had a hunch that alpine's `py-gobject3` package doesn't also include the "GStreamer Good Plug-ins", which contains some of the replaygain analysis functions. I installed the `gst-plugins-good1` package and beet's replaygain functions started working - correctly analysing the files and writing the replaygain tags to them.

In summary, it seems the image needs both the `py-gobject3` and `gst-plugins-good1` added to the list of packages installed by apk in order for ReplayGain to work.
